src/modules/popupWindow.js

```diff
@@ -0,0 +1,26 @@
+import fetchObjDetails from './fetchObjDetails.js';
+import commentPage from './commentPage.js';
+import sendData from './postComment.js';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Kindly avoid disabling linter checks in line. They represent coding standards you share with your team and most of the time enforce best coding practices.